src/CommandLineIface.cpp

```diff
@@ -1,12 +1,37 @@
 #include "CommandLineIface.h"
 #include <QFile>
+#include <QProcessEnvironment>
 
 #include <array>
 
 // Progress bar taken from https://stackoverflow.com/questions/14539867/how-to-display-a-progress-indicator-in-pure-c-c-cout-printf
 #define PBSTR "############################################################"
 #define PBWIDTH 60
 
+namespace {
+    void checkAppleSandbox(QCommandLineParser const &parser) {
+        QProcessEnvironment env(QProcessEnvironment::systemEnvironment());
+        if (!env.contains("APP_SANDBOX_CONTAINER_ID"))
+            return;
+
+        QString command("translateLocally");
+
+        if (parser.isSet("m"))
+            command += QString(" -m %1").arg(parser.value("m"));
```

Comment:
Why is the model affected by the sandboxing? Or, alternatively, why are other options such as listing, downloading and deleting not included here?

Reply:
This bit is only called when `-i` or `-o` are used and cause an error. Since both these options require `-m` to be specified I found it would make sense to include `-m` in the suggested alternative command as well.